### PR TITLE
Added a note about error warnings

### PR DESCRIPTION
**Pull request by RichardLitt**
To cut down on issues that should be in other implementation repos.

### CODE REVIEW

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ There are many ways to contribute to IPFS development. This document will flesh
 
 ### Help with the design
 
-If you find an error or have a suggestion for the overall design of IPFS, please submit an [Issue](https://github.com/ipfs/ipfs/issues/new).
+If you find an error or have a suggestion for the overall design of IPFS, please submit an [Issue](https://github.com/ipfs/ipfs/issues/new). Note: If you an error with an [implementation](README.md#implementations), file an issue in the respective repository. This repository's focus is the abstracted design.
```
Comment (jbenet) [inline CONTRIBUTE.md]: almost, the "abstracted design" is over at https://github.com/ipfs/specs -- this is more of a "landing repo" and "big update place"

**Comment by RichardLitt** [comment]
@jbenet That is... really confusing. Why don't you just put the specs in here?

**Comment by jbenet** [comment]
Because 269 people are watching that do not want to receive the massively high volume of comments we make in repos we work on. I'm being mindful of SNR.

**Comment by RichardLitt** [comment]
:+1:

**Pull request merged**